documentation/2.0/content/userguide/tools-config/logging.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Logging"
+date: 2022-02-04T17:18:00-05:00
+draft: false
+weight: 4
+---
+
+The WebLogic Deploy Tooling bas a built-in logging framework based on `java.util.logging`.  Its logging configuration
+is specified in `$WDT_HOME/etc/logging.properties`.  By default, the logging framework writes to both the console and
+a log file.
+
+#### Log File
+By default, WDT tools write their log files to the `$WDT_HOME/logs` directory and the log file name reflects the name of the tool.
+For example, if you run the `validateModel` tool then the log file will be `$WDT_HOME/logs/validateModel.log`.  These
+log files are overwritten each time you run a particular tool so the file only contains the logs from the last tool
```

Comment:
only contains -> contains only (Position of only: Keep only close to that which it modifies to improve the clarity and accuracy of the sentence.)